### PR TITLE
add test branch to PRs

### PR DESCRIPTION
**Pull request by denishamann-mms**
## 📦 Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## ✅ Checklist

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.

### CODE REVIEW

File: .idea/runConfigurations.xml [hunk 1]
```diff
@@ -1,10 +0,0 @@
-<?xml version="1.0" encoding="UTF-8"?>
-<project version="4">
```
Comment (SimonScholz) [inline .idea/runConfigurations.xml:2]: Can you update the .gitignore accordingly please?

**Review by SimonScholz — COMMENTED** [review]
In the long run it would be nice if we would honor the input value of the workflow:

image

Or even add a new field, where the desired branches can be listed.

**Comment by denishamann-mms** [comment]
> 

We can do that, i think it needs a bit more rework (https://docs.github.com/en/actions/learn-github-actions/events-that-trigger-workflows#workflow_dispatch), the "Use workflow from" is only the branch the script is running in. we could pass that into the gradle script I presume. As we also potentially need to collect also PRs against master, we need both branches to be checked for

File: .idea/runConfigurations.xml (hunk 1 above)
Comment (SimonScholz) [reply .idea/runConfigurations.xml:2]: I meant to add this file also to the .gitignore file.

**Comment by denishamann-mms** [comment]
okay, I added a parameter and now one can optionally pass "BASE_BRANCH" into the script and it checks against it

**Review by SimonScholz — APPROVED** [review]

**Pull request merged**